hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterTrash.java

```diff
@@ -189,6 +189,41 @@ public void testMoveToTrashNoMountPoint() throws IOException,
     assertEquals(2, fileStatuses.length);
   }
 
+  @Test
+  public void testMoveToTrashNoMountPointWithKerBoersUser() throws IOException,
```

Comment:
testMoveToTrashNoMountPointWithKerBoersUser -> testMoveToTrasWithKerberosUser?
a. IMO, this case is not related with `NoMountPoint` or not, right?
b. KerBoers -> Kerberos.

Reply:
OK, thank you for your correction.

Reply:
for a , yes